library/src/hiptensor_options.cpp

```diff
@@ -41,7 +41,26 @@ namespace hiptensor
         , mColdRuns(0)
         , mInputFilename("")
         , mOutputFilename("")
+        , mColMajorStrides(true)
```

Comment:
Should maybe initialize this to the existing value of HIPTENSOR_DEFAULT_STRIDES_COL_MAJOR symbol that was set during cmake.